internal/app/init/pkg/network/static.go

```diff
@@ -30,5 +32,26 @@ func StaticAddress(netconf userdata.Device) (err error) {
 		return err
 	}
 
+	// add a gateway route
+	var network *net.IPNet
+	for _, route := range netconf.Routes {
+		_, network, err = net.ParseCIDR(route.Network)
+		if err != nil {
+			log.Printf("failed to parse static route network %s: %+v", route.Network, err)
+			return err
```

Comment:
should it skip over to the next route if it fails to parse one? 

I'm not sure what is the best behavior here, certainly probably pre-validation, but we can't validate everything (e.g. duplicate routes?).